File: templates/Uwp/_composition/MVVMLight/Project.SplitView/ViewModels/ViewModelLocator_postaction.cs
namespace Param_RootNamespace.ViewModels
{
    public class ViewModelLocator
    {
        private ViewModelLocator()
        {
//^^
//{[{
            SimpleIoc.Default.Register<ShellViewModel>();
//}]}
        }

//{[{
        public ShellViewModel ShellViewModel => ServiceLocator.Current.GetInstance<ShellViewModel>();
//}]}
    }
}

File: templates/_composition/MVVMLight/Project.SplitViewVB/ViewModels/ViewModelLocator_postaction.vb
Namespace ViewModels
    Public Class ViewModelLocator
        Public Sub New()
            '^^
            '{[{
            SimpleIoc.[Default].Register(Of ShellViewModel)()
            '}]}
        End Sub
        '{[{

        Public ReadOnly Property ShellViewModel As ShellViewModel
            Get
                Return ServiceLocator.Current.GetInstance(Of ShellViewModel)()
            End Get
        End Property
        '}]}
    End Class
End Namespace
